demo/README.md

```diff
@@ -0,0 +1,225 @@
+# GETTING STARTED WITH KS-NAV - A BASIC TUTORIAL
+
+## Table of Contents
+
+- [GETTING STARTED WITH KS-NAV - A BASIC TUTORIAL](#getting-started-with-ks-nav---a-basic-tutorial)
+  - [Table of Contents](#table-of-contents)
+  - [1 Overview](#1-overview)
+  - [2 A quick tutorial](#2-a-quick-tutorial)
+    - [2.1 Build Linux for ks-nav](#21-build-linux-for-ks-nav)
+    - [2.2 Run ks-nav on the built image](#22-run-ks-nav-on-the-built-image)
+  - [3 Detailed Tools Configurations](#3-detailed-tools-configurations)
+    - [3.1 Configuring kern\_bin\_db and the local database](#31-configuring-kern_bin_db-and-the-local-database)
+    - [3.2 Configuring nav and navweb for caller graph generation](#32-configuring-nav-and-navweb-for-caller-graph-generation)
+
+## 1 Overview
+
+To start using ks-nav, a Linux (or Yocto) build with debug symbols enabled is required. The tool then extracts symbols, functions and any useful data into a PostgreSQL database. 
+
+In this tutorial, you will:
+
+- Build a default branch from Linux - tag v6.6 is used in this tutorial
+- Extract symbols into a local database using ksnav-kdb
+- Run the navigator using ks-nav
+
+## 2 A quick tutorial
```

Comment:
Are there parts in this tutorial that we should refer to other readme files and possibly improve those readmes with more detail?